### ISSUE TITLE
ERROR: this implementation supports no more than 268435456 reads

### ISSUE THREAD

**Issue by Ahahaha3** [chhylp123/hifiasm#240]
I used HIFI and HIC data to assembly a genome(~3Gb), command: hifiasm -o Y_asm -t 70 --h1 ../HIC/*_1.fq.gz --h2 ../HIC/*_2.fq.gz ../01.ccs/HiFireads/*.subreads.fastq.gz. But i can't understand what is causing this error.

**Comment by chhylp123** [comment]
Mostly data issue like this https://github.com/chhylp123/hifiasm/issues/182. You should check if your Hi-C files or HiFi files are corrected and intact.

**Comment by peritob** [comment]
Thanks for developing this great software. Just thought I would contribute to this error as I wasted a couple of hours testing and might help someone else.  You might need to concatenate your fastq files in HIC/ dir. One merged copy of R1 and one of R2. Silly mistake but gives the listed error message.

**Comment by simon-s-tang** [comment]
Just wanted to add to peritob's comment with my own experience - I got the same error when I was incorporating multiple HiC runs in my genome assembly, but it ran fine when I just added commas between the HiC runs. 

E.g. 
_"...--h1 hic_1_R1.fastq.gz hic_2_R1.fastq.gz --h2 hic_1_R2.fastq.gz hic_2_R2.fastq.gz..."_ didn't work, but
_"...--h1 hic_1_R1.fastq.gz,hic_2_R1.fastq.gz --h2 hic_1_R2.fastq.gz,hic_2_R2.fastq.gz..."_ worked fine.

**Comment by chhylp123** [comment]
Yes, hifiasm required commas when you input HiC reads or UL reads.
